subsys/fs/Kconfig

```diff
@@ -28,6 +28,35 @@ config FAT_FILESYSTEM_ELM
 	help
 	  Use the ELM FAT File system implementation.
 
+config FILE_SYSTEM_LITTLEFS
```

Comment:
why are those being added here? they already exist in the same file

Reply:
Rebase conflict didn't get resolved correctly.